pkg/crane/delete.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Google LLC All Rights Reserved.
+// Copyright 2020 Google LLC All Rights Reserved.
```

Comment:
Don't update the copyright year (here and elsewhere). It expresses a weaker claim (asserting copyright since 2020) than leaving it alone (asserting copyright since 2018).

Reply:
Done.